client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/BusinessContactForm/BusinessContactForm.tsx

```diff
@@ -25,8 +25,7 @@ const BusinessContactForm: React.FC = (): JSX.Element => {
         <div>
             <Typography variant='body1' className={formClasses.fieldName}>פרטי איש קשר:</Typography>
             <Grid container className={formClasses.formRow}>
-                <Grid item xs={3} className={classes.detailsItemField}>
```

Comment:
did you remove the class?

Reply:
now i did, 
there is no use in this styles file anymore